src/cloud/benchflow/data-analyses-scheduler/main.go

```diff
@@ -67,7 +67,7 @@ func main() {
 		panic(err)
 	}
 	
-	dat, err = ioutil.ReadFile("./configuration/data-transformer-scripts-configuration.yml")
+	dat, err = ioutil.ReadFile("./configuration/data-transformers.scheduler.configuration.yml")
```

Comment:
You have `transformers_configuration_file_name` configuration variable that can be used here